### PR TITLE
Update zh-CN.yml

### PR DESCRIPTION
**Pull request by ccmywish**
Add translations

### CODE REVIEW

File: config/locales/zh-CN.yml [hunk 1]
```diff
@@ -164,30 +170,30 @@ zh-CN:
...
+        verified_by: 验证
+        secured_by: 安全保护
+        looking_for_maintainers: 招募维护者
       header:
         dashboard: 控制台
-        settings:
-        edit_profile:
+        settings: 设置
+        edit_profile: 编辑个人信息
         search_gem_html: 搜索 Gems&hellip;
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:180]:
```suggestion
        search_gem_html: 搜索 Gems &hellip;&hellip;
```

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -164,30 +170,30 @@ zh-CN:
...
         sign_in: 登录
         sign_out: 退出
         sign_up: 注册
   mailer:
-    confirm_your_email:
-    confirmation_subject:
-    link_expiration_explanation_html:
+    confirm_your_email: 请在发送到您的邮件中，点击链接，确认您的邮箱地址
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:185]:
```suggestion
    confirm_your_email: 请在发送到您的邮件中点击链接，确认您的邮箱地址。
```

File: config/locales/zh-CN.yml [hunk 3]
```diff
@@ -164,30 +170,30 @@ zh-CN:
...
+    confirmation_subject: 请确认您的邮箱地址
+    link_expiration_explanation_html: 请记住，此链接仅在3小时内有效。您可以使用 <a href='https://rubygems.org/email_confirmations/new'>重新发送确认邮件</a> 页面来请求获得一个新链接
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:187]:
```suggestion
    link_expiration_explanation_html: 请记住，此链接仅在 3 小时内有效。您可以用<a href='https://rubygems.org/email_confirmations/new'>重新发送确认邮件</a>页面来请求获得一个新链接
```

Comment (ParadoxV5) [inline config/locales/zh-CN.yml:187]:
```suggestion
    link_expiration_explanation_html: 请记住，此链接仅在 3 小时内有效。您可以用<a href='https://rubygems.org/email_confirmations/new'>重新发送确认邮件</a>页面来请求获得一个新链接。
```

File: config/locales/zh-CN.yml [hunk 4]
```diff
@@ -164,30 +170,30 @@ zh-CN:
...
     email_confirmation:
-      title:
-      subtitle:
-      confirmation_link:
-      welcome_message:
+      title: 邮件确认
+      subtitle: 即将完成！
+      confirmation_link: 确认邮件地址
+      welcome_message: 欢迎来到 RubyGems.org！请访问下面的链接来验证您的电子邮件。
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:192]: Consistency
```suggestion
      title: 邮箱确认
      subtitle: 即将完成！
      confirmation_link: 确认邮箱地址
      welcome_message: 欢迎来到 RubyGems.org ！请访问下面的链接来验证您的邮箱。
```

File: config/locales/zh-CN.yml [hunk 5]
```diff
@@ -164,30 +170,30 @@ zh-CN:
...
     email_reset:
-      title:
-      subtitle:
-      visit_link_instructions:
+      title: 邮件重置
+      subtitle: Hi %{handle}!
+      visit_link_instructions: 您在 RubyGems.org 上更改了您的邮件地址，请访问以下链接重新激活您的帐户。
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:196]: Consistency cont.
```suggestion
      title: 邮箱重置
      subtitle: Hi %{handle}!
      visit_link_instructions: 您在 RubyGems.org 上更改了您的邮箱地址，请访问以下链接重新激活您的帐户。
```

File: config/locales/zh-CN.yml [hunk 6]
```diff
@@ -299,17 +305,17 @@ zh-CN:
...
         transparent_pages: 构建更透明以及便于浏览的项目页面
     data:
-      title:
+      title: RubyGems.org 数据转储
     download:
-      title:
+      title: 下载 RubyGems
     faq:
-      title:
+      title: FAQ
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:312]:
```suggestion
      title: 常问问题
```

File: config/locales/zh-CN.yml [hunk 7]
```diff
@@ -442,14 +448,14 @@ zh-CN:
...
+      title: 领养
       subtitle_html:
     edit:
-      change_avatar:
+      change_avatar: 修改头像
       email_awaiting_confirmation: 请验证你的新邮箱地址 %{unconfirmed_email}
       enter_password: 输入密码
-      hide_email: 在公开的个人资料里面隐藏我的 Email
-      optional_full_name:
+      hide_email: 在公开的个人资料里面隐藏我的邮件
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:457]: No translation is necessary as ‘Email’ is acceptable untranslated. This suggestion is for consistency.
```suggestion
      hide_email: 在公开的个人资料里面隐藏我的邮箱
```

File: config/locales/zh-CN.yml [hunk 8]
```diff
@@ -442,14 +448,14 @@ zh-CN:
...
+      optional_full_name: 可选。将在公共显示
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:458]:
```suggestion
      optional_full_name: 可填。将公开显示
```

File: config/locales/zh-CN.yml [hunk 9]
```diff
@@ -482,7 +488,7 @@ zh-CN:
       released_with_mfa:
       links:
         badge: 徽章
-        bugs: Bug 追踪
+        bugs: Bug
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:491]: For consistency and translation completness
```suggestion
        bugs: Bug 追踪
```

**Review by ParadoxV5 — COMMENTED** [review]
All ‘Gems’ should simply be ‘Gem’, omitting the plural ‘s’ which is not part of the proper noun.

**Comment by ParadoxV5** [comment]
https://github.com/ccmywish/rubygems.org/blob/dba627ed2a406/config/locales/zh-CN.yml#L293-L300
```suggestion
      founding_html: 此项目由 %{founder} 创立于2009年4月，之后有超过 %{contributors}和%{downloads}的贡献。自从 RubyGems 1.3.6 发布以后，本站名称由 Gemcutter 改名为 %{title} ，从而巩固在 Ruby 社区网站中的核心作用。
      support_html: 虽然 %{title} 不是由一个具体的公司来运作，但也有很多帮助我们走了这么远。网站目前的设计、插画和前端开发都由 %{dockyard} 创建。%{github} 也帮助我们轻松地协助和共享代码。本站最初部署于 %{heroku} ，其一流的服务助证了 RubyGems 是整个社区可依赖的可行方案。目前，我们的基础设施托管于 %{aws}。
      technical_html: 关于本站的一些技术方案：100% Ruby。主站是个 %{rails} 应用。Gem 托管于 %{s3}, 由 %{fastly} 服务, 从发布一个新 Gems 到做好安装准备之间的耗时通常只有几秒。更多信息可以在 GitHub 上阅读基于 %{license} 的 %{source_code}。
```

**Comment by ccmywish** [comment]
Hi @ParadoxV5,

❤️👍 Thanks for your extreme patience (and a good temper that makes you restrain anger) in correcting my errors and suggestions. 😳

😔 I'm very sorry that I originally just wanted to change the translation of one word, but when I opened the file, I started translating something else (just on the web page...). I'm sorry about the pr's insufficient rigorousness.

Should I commit every suggestion of yours? (This may lead to many commits) Or I just make one new commit to follow your advice?

**Comment by simi** [comment]
> Should I commit every suggestion of yours? (This may lead to many commits) Or I just make one new commit to follow your advice?

Pick your choice (easiest way for you), we will most likely squash commits during merge. It will make no difference.

**Comment by ParadoxV5** [comment]
> Hi @ParadoxV5,
> 
> ❤️👍 Thanks for your extreme patience (and a good temper that makes you restrain anger) in correcting my errors and providing suggestions. 😳

@ccmywish What do you mean? It was merely an interest in contributing as part of the Ruby community 🧡.

I wasn’t aware of your translation abilities, and neither I’d call my expertise professional. My idea was to send in my suggestions which you or someone else could further proofread 👍.

> 😔 I'm very sorry that I originally just wanted to change the translation of one word, but when I opened the file, I started translating something else (just on the web page...). I'm sorry about the pr's insufficient rigorousness.

Similarly, neither have I expected I’d proofread through the entire data file 🙃.

**Comment by ParadoxV5** [comment]
Such descriptive commit messages ᗡx definitely wanna consider squashing.

**Comment by ccmywish** [comment]
I've adopted most of the suggestions @ParadoxV5, thanks! By using `Co-authored-by:` automatically by GitHub, your contribution is recorded too.

I've cloned the repo to do serious translation work and I'll squash the commits of 'adopted suggestions' to only one commit locally.

Wait for me to translate the whole CN file today.

**Comment by ccmywish** [comment]
@ParadoxV5 @simi 

The translation of the whole file is done.  Please review (or just merge if you don't have time). 🤝

File: config/locales/zh-CN.yml [hunk 10]
```diff
@@ -21,128 +24,128 @@ zh-CN:
...
+      rubygem_scope_info: 该作用范围仅允许对一个 Gem 进行 推送/撤回 以及 对业主的 添加/移除
+      invalid_key: 已作废的 API 密钥不能被编辑。请删除它并创建一个新的 API 密钥。
+    all_gems: 所有 Gem
+    gem_ownership_removed: 对 %{rubygem_name} 的所有权已经被移除（在使用此 API 密钥决定作用范围后）
   clearance_mailer:
     change_password:
-      title:
-      subtitle:
+      title: 修改密码
+      subtitle: 你好啊，%{handle}！
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:120]: The commit uses ‘你’ for ‘你好’ but ‘您’ everywhere else.

File: config/locales/zh-CN.yml [hunk 11]
```diff
@@ -21,128 +24,128 @@ zh-CN:
...
+      latest: 最新的更新
+      latest_title: 最新的 RSS 订阅更新
       migrating_link_text: 迁移
       mine: 我的 Gem
-      my_subscriptions: 我的订阅
-      no_owned_html: 你没有发布过任何 Gem 。可以看看 %{creating_link} 指南，或从 RubyForge %{migrating_link} 个 Gem 。
-      no_subscriptions_html: 你没有订阅任何 Gem，访问 %{gem_link} 订阅一个！
-      title: 控制台
+      my_subscriptions: 订阅
+      no_owned_html: 您还没有发布过任何 Gem。可以看看 %{creating_link} 指南，或从 RubyForge %{migrating_link} 迁移一个 Gem。
```
Comment (ParadoxV5) [inline config/locales/zh-CN.yml:130]: Wouldn’t `{migrating_link}` already include the word ‘迁移’?

**Review by ParadoxV5 — COMMENTED** [review]
很好 👍

File: config/locales/zh-CN.yml (hunk 11 above)
Comment (ccmywish) [reply config/locales/zh-CN.yml:130]: I guess `%{migrating_link}` is just a URL (which only includes the English word 'migrate')，so I think specifying a Chinese verb `迁移` is acceptable.

Comment (ParadoxV5) [reply config/locales/zh-CN.yml:130]: `%{migrating_link}` could also use the translated term even if not currently is. Better ask the programming team to make sure eh?

Comment (ccmywish) [reply config/locales/zh-CN.yml]: I've fixed it now.

**Comment by simi** [comment]
@ccmywish would you mind to rebase please? There were some changes in some translations and I'm not sure if the current vales are still valid in there. I'll merge once conflicts are resolved. Sorry for the delay.

**Comment by ccmywish** [comment]
@simi I've rebased it onto the master branch now.

**Comment by ccmywish** [comment]
@simi I've resolved the newest conflicts now. Could you please merge it?

**Comment by simi** [comment]
> @simi I've resolved the newest conflicts now. Could you please merge it?

Let's :pray: for the CI. Once :green_apple: I'll merge. Thanks for the rebase.

**Pull request merged**